### DISCUSSION TITLE
Can't upgrade to version  & Retention

### DISCUSSION THREAD

**Discussion by jrm360seclab** [Security-Onion-Solutions/securityonion#8071] [Unsupported Versions] [answered]
I see that 2.3.130 is available, when I try to update get the following:

Found that Security Onion 2.3.120 is currently installed.

Looks like you have an updated OS
Generating new repo archive
Let's see if we need to update Security Onion.
Checking to see if there are hotfixes needed
You are already running the latest version of Security Onion.

Anything I should be doing differently?

Also, I would like to know how to keep logs for up to 2 months at a time or even longer. I have only used 20% of my 800G space but I can only go back to May 7th.

**Comment by xfaith** [comment, marked as answer]
1.  I don't think that 2.3.130 is officially available just yet.
2.   Which logs are you looking at specifically?  Zeek/Suricata/PCAP?

If PCAP this could be the issue:  Maximum Files
By default, Stenographer limits the number of files in the pcap directory to 30K to avoid limitations with the ext3 filesystem.

**Reply by jrm360seclab** [reply]
Thank you xfaith for the reply.

1. I'll wait for the official release then.
2.  I think all if that makes sense. I checked in Kibana, I can't go back further than May 25th for any logs. All gives me logs up to May.

**Reply by dougburks** [reply]
What is the output of the following?
```
sudo so-elasticsearch-indices-list | awk '{print $3}' | cut -d- -f3 |grep 2022 | sort -u
```

**Reply by jrm360seclab** [reply]
Thank you for replying:

here are the results:
2022.05.13
2022.05.14
2022.05.15
2022.05.16
2022.05.17
2022.05.18
2022.05.19
2022.05.20
2022.05.21
2022.05.22
2022.05.23
2022.05.24
2022.05.25
2022.05.26
2022.05.27
2022.05.28
2022.05.29
2022.05.30
2022.05.31
2022.06.01
2022.06.02
2022.06.03
2022.06.04
2022.06.05
2022.06.06
2022.06.07

**Reply by dougburks** [reply]
Check your log_size_limit setting and adjust if necessary:
https://docs.securityonion.net/en/2.3/elasticsearch.html#deleting-indices
